### ISSUE TITLE
Fatal crash after 11 chunks

### ISSUE THREAD

**Issue by gjospin** [gjospin/PhyloSift#445]
Not sure what caused the crash yet but here is the error.

Running /share/eisen-z2/gjospin/Phylosift_test/bin/pplacer  -o /share/eisen-z2/gjospin/Corn/Mucilage_metagenome/Rerun/PS_temp/Mucilage_notcorn_2.fastq/treeDir/V278.updated.11.jplace --verbosity 0 -j 8 -c /share/eisen-z2/gjospin/Phylosift_test/share/phylosift/markers/V278.updated "/share/eisen-z2/gjospin/Corn/Mucilage_metagenome/Rerun/PS_temp/Mucilage_notcorn_2.fastq/alignDir/V278.updated.11.fasta"
malformed number (no digits after initial minus), at character offset 709 (before "an, -inf, 1.99999275...") at /share/eisen-z2/gjospin/Phylosift_test/bin/../lib/Phylosift/pplacer.pm line 438.

**Comment by gjospin** [comment]
The placements in the jplace look like the following lines. The decode_json function breaks on the first -nan

"p":
      [[0.0162720507812, 0, -nan, -inf, 1.99999275208],
        [0.00013125, 1, -nan, -inf, 1.99999275208],
        [0.133151872559, 2, -nan, -inf, 1.99999275208],
        [0.00626387695313, 3, -nan, -inf, 1.99999275208],
        [0.0311523925781, 4, -nan, -inf, 1.99999275208],
        [0.0682316699219, 5, -nan, -inf, 1.99999275208],
        [0.0299826782227, 6, -nan, -inf, 1.99999275208]
      ], "nm": [["26335.1.156.278.2.57.179", 1]

**Comment by gjospin** [comment]
Posted an issue on the pplacer github.
We have identified the issue.  Just waiting for a resolution that we could implement.
https://github.com/matsen/pplacer/issues/331

**Comment by gjospin** [comment]
The solution provided by the pplacer guys was successful.
currently rebuilding the marker package using newest versions of taxtastic and pplacer.

**Comment by ramalok** [comment]
Hi, I'm also running into the same problem with the "malformed number", any idea when the solution will be available? Thanks! Ramiro
PS: btw, I run into this problem when using single reads, but not when running contigs from the same dataset

malformed number (no digits after initial minus), at character offset 786 (before "an, -inf, 1.99999275...") at /share/apps/PhyloSift/phylosift_v1.0.0_02/bin/../lib/Phylosift/pplacer.pm line 440

**Comment by gjospin** [comment]
We have implemented the fix with our reference database as well as using a new version of pplacer/taxit.  Those changes should be reflected in the downloadable content soon.
Thank you for your patience.

**Comment by donutbrew** [comment]
Is there any particular type of dataset this happens with (over a certain size, etc)?

**Comment by gjospin** [comment]
This happens when the sequence you are trying to place contains a residue
that did not exist in the alignment used to build the reference tree for
Pplacer.
We have updated the pplacer and taxit binaries in the github repositories
and we have also rebuilt the reference databases with the new binaries.

We are currently testing a release candidate, it should be available for
download from the website soon.
